TestPlan.md

```diff
@@ -177,14 +194,23 @@ The tree view of maven projects should update when any pom.xml is created/modifi
         }
     ```
 2. Trigger a maven command, in the newly created terminal, verify:
-    1. value of env `MAVEN_OPTS` should be the value you just set in settings.
+    1. On windows platform, input `$Env:MAVEN_OPTS` and press `Enter`, the value of env `MAVEN_OPTS` should be the value you just set in settings.
+    2. On mac/linux platform, input `echo $MAVEN_OPTS` and press `Enter`, the value of env `MAVEN_OPTS` should be the value you just set in settings.
 
 ### Guide to Troubleshooting page if error occurs. 
 1. Make sure `mvn` is not in PATH, or you have set wrong `maven.executable.path` in VS Code.
 2. Open VS Code, open a project, execute `Effective POM`, verify:
     1. It shows a message box for the issue and detailed error message.
     2. Click `Learn more`, it previews `Troubleshooting.md` in the editor.
 
+## Pom.xml file completion
+### Dependency completion
+1. In the `<dependencies>` section, type `<` and accept the suggested `dependency` maven snippet.
+2. Press `Ctrl + Space`.
+3. Verify:
+    1. The snippet is correctly inserted.
+    2. The IntelliSense is correctly worked for `<groupId>`, `<artifactId>` and `<version>`.
+

```

Comment:
Remove the empty line here.

Reply:
@Eskibear, I have updated TestPlan.md according to your comment, please have a view. Thanks.